script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -131,32 +158,43 @@ I nod my head in understanding and resume eating.
 [sha:8743b20797717f6430f00a7f6cd0d32df4f91a65]{
 -- Page 335, Offset 14743.
 -- 朝食と昼食を少なめにしていたおかげか、夜になって食欲が戻ってきた。
-Perhaps it was because I had eaten less breakfast and lunch, but it seems like my appetite had come back in the evening.
+Perhaps as a result of having a light breakfast followed by a light lunch,
+ by evening I had developed quite the appetite.
 }
 [sha:d47919b516ce7b6992ea0233941e363a5e1fdaf2]{
 -- Page 335, Offset 14744.
 -- ……それでも、今夜の<献立|メニュー>はやや目に痛い。
-...But with that being said, tonight's menu is rather sore to my eyes.
+// Replaced 目に痛い with EN set phrase
+...though despite that, I'm finding tonight's menu a little difficult to stomach.
 }
 [sha:dbf45b03b93a75bdc5ccb7f057db36dfb3042630]{
 -- Page 335, Offset 14745.
 -- 牛のヒレ肉やフォアグラだけでなく、真っ赤なソースが血液を連想させて、否応にも昨日までの出来事を思い返してしまう。
-It wasn't only the beef fillet and foie gras, but also the bright red sauce that reminded me of blood, which inevitably made me think back to the events of yesterday.
+Not just because of the beef tenderloin and foie gras,
+ but also because the deep red sauce forcefully reminds me of blood,
+ and by extension all of the events that led up to yesterday.
 }
 [sha:fb7862f81594d6d80ddede52bbe998bcbf21f6eb]{
 -- Page 336, Offset 14746.
 -- 突然の殺人。ありえない再会。ありえない怪物。
-The sudden murders. The unlikely reunion. The unbelievable monster.
+A freak murder. An impossible reunion. An unbelievable monster.
 }
 [sha:7ed0fcc21c54df4a379806f95d9ef00b78a77334]{
 -- Page 336, Offset 14747.
 -- 多くの死に背中を押されて伸ばした指と、死の手触り。
-The sensation of death as I stretch my fingers to the back of the body full of death.
+// Definitely check me on this one
+// If the first clause is about a large amount of death bearing down on his back,
+// as I've interpreted it here, then I've adjusted death to mortality so that the
+// sentence doesn't read as clunkily.
+The heavy weight of my own mortality pressing down on my back,
```

Comment:
I'm not so sure about this translation. Literally its like "The elongated finger(s) that pushed me towards a lot of death and the touch of death." and I think its more like "the outstretched finger [of fate]" or something rather than _his_ outstretched fingers. I don't think the interpretation of the last clause is right at all tbh. It's more like "a brush with death" or "the touch of death" or something like that. There's also that strange passive tense of 背中を押されて that you could argue very strongly isn't necessary but it is missing. 

I don't think this reads particularly well, but hopefully it illustrates what I'm saying
> 多くの死に背中を押されて伸ばした指と、死の手触り。
The elongated finger that saw me pushed toward those heavy casualties and the the touch of death.

Reply:
Yeah, this is the line I was sure was wrong. I'll sleep on this interpretation of it, but hard to fully grok what he's getting at so that I can write it correctly =/